integrations/helm/release/release.go

```diff
@@ -352,6 +353,23 @@ func values(corev1 k8sclientv1.CoreV1Interface, ns string, valuesFromSource []fl
 				}
 				return result, fmt.Errorf("unable to yaml.Unmarshal %v from URL %s", b, url)
 			}
+		case v.ChartFileRef != nil:
+			cf := v.ChartFileRef
+			filePath := cf.Path
+			optional := cf.Optional != nil && *cf.Optional
+			f, err := readLocalFile(fmt.Sprintf("%s/%s", chartPath, filePath))
```

Comment:
I think it would be cleaner to change this to `readLocalChartFile(chartPath, filePath)`, and move the construction of the path there (so it actually does what the comment tells it does).

`// readLocalFile attempts to read a file from the chart path.`

Reply:
```suggestion
			f, err := readLocalFile(filepath.Join(chartPath, filePath))
```